libs/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart-configuration.service.ts

```diff
@@ -70,7 +70,7 @@ export class FhirChartConfigurationService {
     }
   }
 
-  setSummaryRange(months: number) {
+  showSummaryRange(months: number) {
```

Comment:
These method names are confusingly similar. Is there a situation where we need to set the summary range but not show it? If not, it might make more sense to have a single `setSummaryRange` method that turns the summary range off if `months` is 0.